composer.lock

```diff
@@ -877,20 +1358,6 @@
                 "MIT"
             ],
             "description": "PHPStan - PHP Static Analysis Tool",
-            "funding": [
```

Comment:
You're probably using an older version of Composer that doesn't support this type of metadata. Please make sure it's not removed by mistake.

Reply:
Thanks for the tip, I'll try again.

Reply:
I updated, and it's still removed, but if you look at the diff, it is also added in the `xdebug-handler` package metadata above.

Reply:
Yeah, this is because Psalm requires it (unlike PHPUnit!). I'm fine with the changes in the lock file, just wanted to make sure it's not a mistake.